### PR TITLE
Merge job fixes

### PR DESCRIPTION
**Pull request by hmlanigan**
Fix Licenses verification in the Makefile and in the files, some capitalization of characters needed.  As well as allowing LICENSE.ricochet2200, from some du and winery files.

Update the Makefile to install dependencies: bzr, zip, mongodb and go 1.12.  Change to go 1.12 necessitated changes based on go fmt and "go tool vet" becomes "go vet"

### CODE REVIEW

**Comment by hmlanigan** [comment]
https://jenkins.juju.canonical.com/job/github-check-merge-juju-utils/5/console with change to job itself also:
#sudo snap install go --channel=1.10/stable --classic
make install-dependencies

looking at:
sudo DEBIAN_FRONTEND=noninteractive apt-get install zip bzr juju-mongodb3.2 --yes

**Comment by hmlanigan** [comment]
!!build!!

**Comment by hmlanigan** [comment]
https://jenkins.juju.canonical.com/job/github-check-merge-juju-utils/7/console with
#sudo DEBIAN_FRONTEND=noninteractive apt-get install zip bzr juju-mongodb3.2 --yes

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,7 @@ check: check-licence check-go
 check-licence:
 	@(fgrep -rl "Licensed under the LGPLv3" .;\
 		fgrep -rl "MACHINE GENERATED BY THE COMMAND ABOVE; DO NOT EDIT" .;\
+		fgrep -rl "license that can be found in the LICENSE.ricochet2200 file" .; \
```
Comment (achilleasa) [inline Makefile]: Isn't this a regex? Does the '.' after LICENSE need to be escaped?

Comment (hmlanigan) [reply Makefile]: fgrep = grep -F
"Interpret PATTERNS as fixed strings, not regular expressions."

Comment (hmlanigan) [reply Makefile]: fgrep is also deprecated, changing all to 'grep -F'

**Comment by hmlanigan** [comment]
$$merge$$

**Pull request merged**